examples/src/intTest/java/ca/nrc/cadc/UserStorageBrowserPage.java

```diff
@@ -733,8 +733,9 @@ public PermissionsFormData getValuesFromEditIcon() throws Exception
     {
         WebElement editIcon = find(xpath("//span[contains(@class, 'glyphicon-pencil')]/a"));
 
-        return new PermissionsFormData(editIcon.getAttribute("data-readGroup"),
-                                       editIcon.getAttribute("data-writeGroup"));
+        return new PermissionsFormData(editIcon.getAttribute("readGroup"),
+                editIcon.getAttribute("writeGroup"),
+                editIcon.getAttribute("publicPermissions"));
```

Comment:
The publicPermission attribute is never used and caused the tests to fail initially.  Also, the attributes are now data-XXX attributes for accuracy and compliance, so revert these changes please.